core/monitor/price/snapshot.go

```diff
@@ -177,12 +177,15 @@ func (e *Engine) serialisePriceRanges() []*types.PriceRangeCache {
 	sort.SliceStable(prc, func(i, j int) bool {
 		if prc[i].Bound.UpFactor.Equal(prc[j].Bound.UpFactor) {
 			if prc[i].Bound.DownFactor.Equal(prc[j].Bound.DownFactor) {
-				return prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon && prc[i].Bound.Active == prc[j].Bound.Active
+				if prc[i].Bound.Trigger.Horizon == prc[j].Bound.Trigger.Horizon {
+					return prc[i].Bound.Active == prc[j].Bound.Active
+				}
+				return prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon
 			}
-			return prc[j].Bound.DownFactor.LessThan(prc[i].Bound.DownFactor) && prc[i].Bound.Active == prc[j].Bound.Active
+			return prc[j].Bound.DownFactor.LessThan(prc[i].Bound.DownFactor)
 		}
 
-		return prc[j].Bound.UpFactor.GreaterThan(prc[i].Bound.UpFactor) && prc[i].Bound.Active == prc[j].Bound.Active
+		return prc[j].Bound.UpFactor.GreaterThan(prc[i].Bound.UpFactor)
```

Comment:
not really important but why reverse sort rather than return LessThan like everywhere?

Reply:
IDK, this is what was there

Reply:
so was the bug :)